src/seedu/addressbook/commands/JunkCommand.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.commands;
+
+/**
+ * Shows help instructions.
+ */
+public class JunkCommand extends Command {
+
+    public static final String COMMAND_WORD = "junk";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": THIS IS A LINE OF UNNECESSARY JUNK. YOU JUST WASTED 1 MIN OF YOUR LIFE =)\n"
```

Comment:
Take note of indentation! Always try to keep the code neat and readable :) Try to avoid lines longer than 110 chars.